remake.yml

```diff
@@ -0,0 +1,28 @@
+sources:
+  - 1_fetch/src/fetch.R
+  - 2_process/src/process.R
+  # trying to trick the bot since I've already done it right
+  - 3_plot/src/plot.R
+  - 4_diagnostics/src/diagnostics.R
+
+targets:
+  all:
+    depends:
+      - 3_plot/out/figure_1.png
+      - 4_diagnostics/out/model_diagnostic_text.txt
```

Comment:
There are a few file targets that won't currently be built using the default target (i.e., if you just call `scmake()` with no arguments) - could you add those here?

Reply:
I think I've resolved this now. I think I just did these b/c I knew the other products depended on these.